types/wordpress__edit-post/package.json

```diff
@@ -1,15 +1,17 @@
 {
     "private": true,
     "name": "@types/wordpress__edit-post",
-    "version": "7.5.9999",
+    "version": "8.4.9999",
     "projects": [
         "https://github.com/WordPress/gutenberg/tree/master/packages/edit-post/README.md"
     ],
+    "minimumTypeScriptVersion": "5.0",
```

Comment:
why is this needed?

Reply:
Because this check fails:
https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/10161564814/job/28100341574

Reply:
@sheetalkamat

Reply:
Ping @sheetalkamat